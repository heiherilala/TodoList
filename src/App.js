import ToDoList from "./pages/ToDoList";
import "./styles.css";


export default function App() {
  return (
    <>
      <ToDoList />
    </>
  );
}
