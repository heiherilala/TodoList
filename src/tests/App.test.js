import renderer from "react-test-renderer";
import '@testing-library/jest-dom/extend-expect';
import App from "../App";

it("have to be same as the Snapshot", () => {
    const component = renderer.create(<App />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });