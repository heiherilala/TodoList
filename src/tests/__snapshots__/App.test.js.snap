// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`have to be same as the Snapshot 1`] = `
<div
  className="container todo-container"
>
  <div
    className="row"
  >
    <div
      className="to-do-table d-flex flex-column justify-content-start col-sm-5 card bg-secondary"
    >
      <div
        className="card-header text-white"
      >
         To Do 
      </div>
      <form
        onSubmit={[Function]}
      >
        <input
          className="w-100"
          onChange={[Function]}
          placeholder="Write a 'to do' "
          type="text"
          value=""
        />
      </form>
    </div>
    <div
      className="to-do-table d-flex flex-column justify-content-start col-sm-5 card bg-secondary"
    >
      <div
        className="card-header text-white"
      >
         Done 
      </div>
      <form />
    </div>
  </div>
</div>
`;
